<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="BA-000235932" name="com.telus.wnp.tests.CheckCatchException" tests="1" failures="1" timestamp="2022-05-02T17:59:31 IST" time="75.466" errors="0">
  <testcase name="testMethod_ForeignToTelusPortin" time="75.466" classname="com.telus.wnp.tests.CheckCatchException">
    <failure type="java.lang.AssertionError" message="java.lang.AssertionError: Exception Occured: Unable to verify the status of the Subscriber from Smart desktop expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: java.lang.AssertionError: Exception Occured: Unable to verify the status of the Subscriber from Smart desktop expected [true] but found [false]
at com.test.ui.actions.Validate.assertTrue(Validate.java:1911)
at com.test.ui.actions.Validate.assertTrue(Validate.java:1924)
at com.telus.wnp.steps.SmartDesktopSearchPageSteps.verifySubscriberStatusFromSD(SmartDesktopSearchPageSteps.java:75)
at com.telus.wnp.tests.CheckCatchException.testMethod_ForeignToTelusPortin(CheckCatchException.java:123)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
Caused by: java.lang.AssertionError: Exception Occured: Unable to verify the status of the Subscriber from Smart desktop expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertTrue(Assert.java:45)
at com.test.ui.actions.Validate.assertTrue(Validate.java:1907)
... 18 more
]]>
    </failure>
  </testcase> <!-- testMethod_ForeignToTelusPortin -->
  <system-out/>
</testsuite> <!-- com.telus.wnp.tests.CheckCatchException -->
